webpack.prod.config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path')
+const {merge} = require('webpack-merge')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const CopyPlugin = require("copy-webpack-plugin");
+const baseWebpackConfig = require('./webpack.base.config')
+
+const prodWebpackConfig = merge(baseWebpackConfig, {
+	// BUILD settings gonna be here
+	mode: 'production',
+	output: {
+		path:path.resolve(__dirname, "./prod"),
+		filename: 'bundle.js',
+	},
```

Comment:
```suggestion
	output: {
		path:path.resolve(__dirname, "./dist"),
		filename: 'bundle.js',
	},
```
Перемести это в основной конфиг и используй одну папку для дев и прод